src/shared/killall.c

```diff
@@ -97,21 +96,21 @@ static bool ignore_proc(const PidRef *pid, bool warn_rootfs) {
         if (uid != 0)
                 return false;
 
+        (void) pidref_get_comm(pid, &comm);
```

Comment:
isn't this under the control of the process being killed?

Reply:
But it's limited to root? Also this isn't security sensitive I'd say, since for some errors we just return true.

Reply:
Can the existing mechanism with the `@` be used instead, so that the same thing is used everywhere?

Reply:
`sd-clone` uses `clone()` directly, and to pass this bit of information is kinda hard. (The explanation is included in the commit msg)